### ISSUE TITLE
Show disabled specs

### ISSUE THREAD

**Issue by wulftone** [bradphelan/jasminerice#104]
It would be great to show `xit` and `xdescribe` when running a suite, so we know what specs we've forgotten about.  : )  What do you think?

**Comment by jejacks0n** [comment]
It requires changes to Jasmine (which is just a dependency of this project).  So it's non-trivial or easy to maintain between versions. Side story: I tried to get this into the Jasmine project (being a former pivotal labs employee) but nothing came of it.

In the [Teabag project](https://github.com/modeset/teabag) I do a little hack to force "it" calls with no functions to behave as though they're pending, but adding that functionality to xit/xdescribe isn't a friendly hack. Mocha has support for this, and I've been pretty happy using mocha (teabag allows jasmine, mocha, or qunit).

TLDR; no, it can't really be added in this project since Jasmine's just a dependency.
